watcher_prover/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM elixir:1.14.5-otp-25
+
+# Copy watcher_prover inside docker
+WORKDIR /watcher
+COPY . /watcher
+
+# Install cago
+RUN apt-get -qq update
+RUN apt-get install -y -q build-essential curl
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
We may want to fix the rust version on installation so we don't run into trouble when a new rust version is released. We need to ask the devs about it.

Reply:
We'll leave this be, and fix it later.